### PR TITLE
Remove hard-coded `Precomputed_values.compiled = None`

### PR DESCRIPTION
**Pull request by mrmr1993**

### CODE REVIEW

**Comment by mrmr1993** [comment]
!ci-build-me

File: src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml [hunk 1]
```diff
@@ -1668,18 +1668,7 @@ let snark_hashes =
       let json = Cli_lib.Flag.json in
       let print = Core.printf "%s\n%!" in
       fun () ->
-        let hashes =
```
Comment (georgeee) [inline src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml:1671]: Why keep a command that only prints `[]`/`<empty>`?

Even is we want to keep it, let's simplify the code down to:

```
if json then Core.print "[]\n"
```

**Review by georgeee — CHANGES_REQUESTED** [review]

**Comment by mrmr1993** [comment]
!ci-build-me

File: src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml [hunk 2]
```diff
@@ -1667,21 +1667,7 @@ let snark_hashes =
...
-                     (* Throw away the constraint system ID to avoid changing the
-                        format of the output here.
-                     *)
-                     Md5.to_hex digest )
-          | None ->
-              []
-        in
-        if json then print (Yojson.Safe.to_string (Hashes.to_yojson hashes))
-        else List.iter hashes ~f:print]
+      fun () -> if json then print "[]\n"]
```
Comment (georgeee) [inline src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml:1670]:
```suggestion
      fun () -> if json then Core.printf "[]\n"]
```
And remove the line above

**Review by georgeee — APPROVED** [review]

**Pull request merged**